clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -11019,38 +11014,24 @@ void LinkerWrapper::ConstructJob(Compilation &C, const JobAction &JA,
       if (TC->getTriple().isSPIROrSPIRV() &&
           TC->getTriple().getSubArch() == llvm::Triple::NoSubArch) {
         TargetTriple = TC->getTriple();
-        break;
+        SmallVector<std::string, 8> SYCLDeviceLibs;
+        auto IsSPIR = TargetTriple.isSPIROrSPIRV();
+        bool IsSpirvAOT = TargetTriple.isSPIRAOT();
```

Comment:
nit - I know this was in the old code - but it is odd to use `bool` and `auto` next to each other when they are basically the same usage.